roaring/naive.go

```diff
@@ -0,0 +1,321 @@
+// Copyright 2017 Pilosa Corp.
```

Comment:
this date seems wrong

Reply:
it's the date in the licenses for pretty much all the files. @jaffee should we update all our licenses?

Reply:
afaict this is fine. we may want to update everything to the current year at some point, but it doesnt seem to matter